docs/adr/0004-return-uninflected-va-keys.md

```diff
@@ -0,0 +1,25 @@
+# 4. Return Uninflected VA Keys in vets-api
+
+Date: 2020-10-30
+
+Whom:
+- Philip Becker
+
+## Status
+
+Accepted
+
+## Context
+
+Using the Rails 6 zeitwerk file loader requires strict naming conventions of files.  So without a Rails inflection, `va_something` class files are expected to have the class name of `VaSomething`.  With an inflection, that file would be the expected `VASomething`.  Using this inflection has a downstream affect on json responses transformed into camelCase by the [olive_branch](https://github.com/vigetlabs/olive_branch) gem.
```

Comment:
```suggestion
Using this inflection has a downstream affect on the named attributes in serializes and thus, json responses transformed into camelCase by the [olive_branch](https://github.com/vigetlabs/olive_branch) gem.
```

Reply:
how can i not take this opportunity to include the word `thus`!?